derive/src/to_delegate.rs

```diff
@@ -449,17 +449,17 @@ pub fn generate_where_clause_serialization_predicates(
 			// add json serialization trait bounds
 			if client {
 				if visitor.server_to_client_type_params.contains(&ty.ident) {
-					bounds.push(parse_quote!(_serde::de::DeserializeOwned))
+					bounds.push(parse_quote!(jsonrpc_core::serde::de::DeserializeOwned))
```

Comment:
please use `#core_name` instead of `jsonrpc_core`. This allows using:
```
rpc = { package = "jsonrpc-core", ... }
```
inside the `Cargo.toml` - so the crate is renamed, and in such case your `jsonrpc_core` import would not work.

Note that the code we generate here is running in the context of the user's crate, so we don't want to enforce them to import `jsonrpc-core` in a specific way.
```suggestion
					bounds.push(parse_quote!(#core_name::serde::de::DeserializeOwned))
```

Reply:
I think it might be better to have `_jsonrpc_core` instead of `#core_name`, as the name is already imported as `_jsonrpc_core` and all other parts of the code are using `_jsonrpc_core` to refer to the crate.